src/common/TokenParser.ts

```diff
@@ -117,7 +113,7 @@ export class TokenParser {
 
     private convertSymbol(symbol: string): string {
         if (symbol.startsWith("0x")) {
-            return Config.web3.utils.hexToAscii(symbol);
+            return Config.web3.utils.hexToAscii(symbol).replace(/\u0000*$/, "");
```

Comment:
what's `u0000*$`?

Reply:
Removes <img width="1324" alt="screenshot 2018-01-26 20 14 00" src="https://user-images.githubusercontent.com/3277207/35468564-826b2f9e-02d5-11e8-8de1-dcf9687f2417.png"> Example of trx https://api.trustwalletapp.com/transactions/0x34aaa2068d7a9609af5f8618aa83ab0129985f794ebf05d3d1be3c876e1c78d5